src/backend/preprocessor/cfgGenerator/For.js

```diff
@@ -0,0 +1,48 @@
+define([
+    '../Cfg'
+], function (Cfg) {
+    var cfgGenerator;
+    
+    function For(paramNode, options) {
+	var body = cfgGenerator(paramNode.body[0], options);
```

Comment:
could you please name these constructs (paramNode.body[0]) - introduce a helper variable?